contractcourt/htlc_timeout_resolver.go

```diff
@@ -529,6 +530,7 @@ func (h *htlcTimeoutResolver) handleCommitSpend(
 				Fee: sweep.FeePreference{
 					ConfTarget: sweepConfTarget,
 				},
+				Force: true,
```

Comment:
@Crypt-iQ pointed out that the second level spend has already confirmed at this point.

Reply:
Yeah this is unnecessary.